src/arrayMethodSort.js

```diff
@@ -4,9 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Using `[].__proto__` to define a method on the array prototype is not recommended. Instead, consider using `Array.prototype` directly to add the `sort2` method. This is a more standard and safer approach.